### DISCUSSION TITLE
Github Enterprise Server API: List Security Advisories

### DISCUSSION THREAD

**Discussion by ajelic-cookieai** [community/community#59398] [Enterprise]
### Select Topic Area

Question

### Body

Does REST API or GraphQL API for Github Enterprise Server have an endpoint for listing Security Advisories for a repository?
When making the same request as for [Github REST API](https://docs.github.com/en/rest/security-advisories/repository-advisories?apiVersion=2022-11-28#list-repository-security-advisories) on Github Enterprise Server it fails with HTTP error `406 Not Acceptable`. Could it be that the Accept header (`Accept: application/vnd.github+json`) is invalid?

**Comment by byrneh** [comment]
The link you posted was for the Github "Feee, Pro and Team" version on GitHub .com.
To check what API endpoints are availabe for your self-hosted version of Enterpirse server select that version from the documentaion [Version] drop down selector.
Illustrated here https://docs.github.com/en/get-started/learning-about-github/about-versions-of-github-docs  You need to remember to this when checking REST, GraphQL or other product information

I do not see the "List repository security advisories" you metion avaiable for any version fo Enterprise Server on the documentation.
